ironfish/src/rpc/routes/wallet/postTransaction.ts

```diff
@@ -35,6 +36,10 @@ router.register<typeof PostTransactionRequestSchema, PostTransactionResponse>(
   PostTransactionRequestSchema,
   async (request, node): Promise<void> => {
     const account = getAccount(node, request.data.account)
+    Assert.isNotNull(
+      account.spendingKey,
+      'Account must have spending key to post a transaction',
+    )
```

Comment:
Can we just move this into node.wallet.post so it doesn't need to be done everywhere?